app/src/main/java/waiter/Connectable/ClientConnection.java

```diff
@@ -21,13 +21,13 @@ public ClientConnection(Socket socket) throws IOException {
     }
 
     public String read() throws IOException {
-        StringBuilder requestStartLineBuilder = new StringBuilder();
-        while(notEndOfLine(requestStartLineBuilder)) {
-            requestStartLineBuilder.append((char) this.bufferedReader.read());
+        StringBuilder stringBuilder = new StringBuilder();
+        while(notEndOfLine(stringBuilder)) {
+            stringBuilder.append((char) this.bufferedReader.read());
         }
 
-        String requestStartLine = requestStartLineBuilder.toString();
-        return requestStartLine.strip();
+        String readInString = stringBuilder.toString();
+        return readInString.strip();
```

Comment:
Consider inlining this to:
```java
return stringBuilder.toString().strip();
```

Reply:
Addressed in [this commit](https://github.com/jonmanahan/waiter/pull/17/commits/57edb196601a5e57e7dc93da5576472860a385da)